fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/SkolemNodeRdfContext.java

```diff
@@ -93,6 +94,10 @@ public SkolemNodeRdfContext(final FedoraResource resource,
             return null;
 
         } catch (final RepositoryException e) {
+            if (e instanceof ItemNotFoundException) {
+                // Effectively skip resources that do not exist
```

Comment:
I'm not sure I totally understand the concern here. @awoods is now removing links from resources to deleted resources, but those links were inaccessible from either the side of the link. @ajs6f have your concerns been addressed?